src/components/AuthLogin/UabcHeaderText.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const UabcHeaderText = () => {
+  return (
+    <div className="text-center">
+      <p className="font-proxima text-[4rem] font-bold leading-[2.5rem] text-primary">
+        UABC
+      </p>
+    </div>
+  );
+};
+
+export default UabcHeaderText;
```

Comment:
components should be named exports
```suggestion
import React from "react";

export const UabcHeaderText = () => {
  return (
    <div className="text-center">
      <p className="font-proxima text-[4rem] font-bold leading-[2.5rem] text-primary">
        UABC
      </p>
    </div>
  );
};
```